CenterForOpenScience/osf.io
# Fix/DELETE CollectionLinkedRegistrationsRelationship 500 Error [EMB-571]

## Purpose

You get a 500 with a DELETE request to the CollectionLinkedRegistrationsRelationship -
 
## Changes

Have CollectionLinkedRegistrationsRelationship inherit from CollectionLinkedNodesRelationship instead of LinkedRegistrationsRelationship and the CollectionMixin.  That view contains everything it needs, besides the overrides in the view itself (allows it to call Collection.remove_object` instead of Collection.perform_destroy which isn't a thing. 


## QA
None needed - but a DELETE request http://localhost:8000/v2/collections/9jdgr/relationships/linked_registrations/ with request body:
```
{
	"data": [
		{"id": "s6teh", "type": "linked_registrations"}
	]
}
``` 
should succeed.

## Documentation



## Side Effects



## Ticket

https://openscience.atlassian.net/browse/EMB-571
